src/routes/manage_job/get.rs

```diff
@@ -48,6 +61,45 @@ pub struct JobStatusResponseBody {
 	job_status: ValidStatus,
 }
 
+async fn job_result(
+	job_id: i32,
+	req: JobResultRequest,
+	conn_pool: Pool<Postgres>,
+) -> Result<impl warp::Reply, warp::Rejection> {
+	let query = sqlx::query!(
+		r#"
+		SELECT result FROM email_results
+		WHERE job_id = $1
```

Comment:
One requirement in the spec is that the endpoint should error if the job is not completed.

Let me know how you think this is best completed. Maybe your initial idea of adding a `status` column in the `job` table was a good one

Reply:
Small up on this requirement

Reply:
After discussing offline, let's do in a separate PR